packages/emulation/libretro-vecx/package.mk

```diff
@@ -15,7 +15,11 @@ PKG_LIBPATH="${PKG_LIBNAME}"
 PKG_LIBVAR="VECX_LIB"
 
 make_target() {
-  make
+  if [ "$OPENGLES_SUPPORT" = yes ]; then
+    HAS_GLES=1 make
+  else
+    make
+   fi
```

Comment:
^ please fix the indentation. Can you also tweak the commit messaage (not PR title) to `game.libretro.vecx: use GLES when GL is not available` as it better describes the change now. Thx.